### PR TITLE
Superhero JWT with iframe wallet

### LINKED ISSUE #168: Wallet detector was not find a wallet in parent if wallet is an iframe
For example when the app is running in Firefox 79.0 inside the iframe at superhero-ui there is one issue and one feature request:
1. There is no popup with conformation. Fix that and make sure that sdk wallet detector works with parent(superhero-ui) in the same way as with the extension.
3. Add jitsi domains to the white list to enable the login without the confirmation.

### PR DESCRIPTION
**Pull request by shapkarin**
### Test cases

1:
- open superhero-ui meet tab and prejoin screen (http://localhost:8080/meet/test)
- auth with web wallet at right sidebar
- you will see the popup about JWT to the Superhero-jitsi
- confirm and get success auth to jitsi

2:
- open main page
- auth with web wallet at right sidebar
- auth with popup at superhero-ui
- go to the meed and create a room

related issue https://github.com/aeternity/jitsi-meet/issues/168

### CODE REVIEW

File: react/features/base/connection/functions.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ export function getCurrentConferenceUrl(stateful: Function | Object) {
  */
 export function getInviteURL(stateOrGetState: Function | Object): string {
     if (isInIframe()) {
-        return document.referrer;
+        // TODO: fix error URI is undefined
+        return 'http://localhost:8081/meet/asdasdasd';//document.referrer;
```
Comment (Chima1707) [inline react/features/base/connection/functions.js]: Please can you explain what is happening here?

Comment (shapkarin) [reply react/features/base/connection/functions.js]: I open it at incognito with Firefox (without extension) and got an error. As I think it means that `document.referrer` is undefined in some reason.
Снимок экрана 2020-08-18 в 17 28 40

Comment (shapkarin) [reply react/features/base/connection/functions.js]: as I see an error about that line https://github.com/aeternity/jitsi-meet/blob/075de8dec090bec0b04196cef86320cf31f25171/react/features/base/premeeting/components/web/CopyMeetingUrl.js#L159

Comment (shapkarin) [reply react/features/base/connection/functions.js]: related issue: https://github.com/aeternity/jitsi-meet/issues/111#issuecomment-675671943

Comment (venimus) [reply react/features/base/connection/functions.js]: yeah `document.referer` is reasonable to be undefined in incognito

Comment (shapkarin) [reply react/features/base/connection/functions.js]: @venimus not actually. I researched about problem and in FF it's not `undefined` but just a `domain name` without any path. That's related to the some browser security policy. But not sure that can be solved just with some `<meta>` or something.

Comment (shapkarin) [reply react/features/base/connection/functions.js]: @venimus you can check that comment with explanation https://github.com/aeternity/jitsi-meet/issues/111#issuecomment-675743340 and that link https://support.mozilla.org/en-US/questions/1067835

**Review by DeeJayElly — APPROVED** [review]

File: react/features/base/connection/functions.js [hunk 2]
```diff
@@ -41,7 +41,8 @@ export function getCurrentConferenceUrl(stateful: Function | Object) {
  */
 export function getInviteURL(stateOrGetState: Function | Object): string {
     if (isInIframe()) {
-        return document.referrer;
+        // TODO: fix error URI is undefined
+        return 'http://localhost:8081/meet/test90003000';//document.referrer;
```
Comment (shapkarin) [inline react/features/base/connection/functions.js]:
```suggestion
        return document.referrer;
```

File: react/client.js [hunk 3]
```diff
@@ -1,22 +1,86 @@
...
+            detector.stopScan();
+            await client.connectToWallet(await foundWallet.getConnection());
+            await client.subscribeAddress('subscribe', 'current');
+            cb();
+        }
+    });
+};
+
+async function initStaticClient () {
+    staticClient = true;
```
Comment (venimus) [inline react/client.js]: where is this used?

**Comment by mradkov** [comment]
I tried it and it works ok on local setup.

File: react/client.js [hunk 4]
```diff
@@ -1,22 +1,86 @@
...
+        connectionInfo: { id: 'spy' }
+    });
+
+    // eslint-disable-next-line new-cap
+    const detector = await Detector({ connection });
+
+    detector.scan(async ({ wallets, newWallet }) => {
+        const foundWallet = newWallet || Object.values(wallets)[0];
+
+        console.log({ foundWallet });
```
Comment (venimus) [inline react/client.js]: i think this should be removed

Comment (shapkarin) [reply react/client.js]: sure

**Review by venimus — APPROVED** [review]

**Pull request merged**